aptos-move/framework/src/natives/mod.rs

```diff
@@ -225,6 +232,23 @@ pub fn all_natives(
     make_table_from_iter(framework_addr, natives)
 }
 
+pub fn all_test_natives(framework_addr: AccountAddress) -> NativeFunctionTable {
+    let mut natives = vec![];
+
+    macro_rules! add_natives_from_module {
+        ($module_name: expr, $natives: expr) => {
+            natives.extend(
+                $natives.map(|(func_name, func)| ($module_name.to_string(), func_name, func)),
+            );
+        };
+    }
+
+    // Dummy natives.
```

Comment:
So these are no lunger dummy natives right, they are actual, useful test natives we want to have!